File: ChatApp.Domain/Entities/ChatParticipant.cs
namespace ChatApp.Domain.Entities
{
    public class ChatParticipant
    {
        public long ID { get; set; }
        public long ChatID { get; set; }
        public long UserID { get; set; }
        public string Role { get; set; }
        public DateTime JoinedAt { get; set; }
    }
}

File: Frontend/src/types/chat-participant.ts
import { ChatParticipantType } from './enums';

export interface IChatParticipant {
  id: number;
  chatID: number;
  userID: number;
  fullName: string;
  avatar: string;
  role: ChatParticipantType;
  joinedAt: Date;
}
